src/app/cli/src/app.rs

```diff
@@ -337,6 +337,9 @@ fn configure_database_components(
     catalog_builder: &mut CatalogBuilder,
     db_configuration: &DatabaseConfiguration,
 ) -> Result<(), InternalError> {
+    // TODO: Remove after adding implementation of FlowEventStore for databases
+    catalog_builder.add::<kamu_flow_system_inmem::FlowEventStoreInMem>();
```

Comment:
At first glance, we don't need this for persistence storage of accounts, but in general, it doesn't seem difficult to implement right now

Reply:
What is not going to be diffcult? A repo for flow system would be quite hard. There are many complex queries to implement.